README.md

```diff
@@ -37,43 +37,51 @@ at <a href="https://docs.snowflake.net/manuals/user-guide/nodejs-driver.html">No
 Test
 ======================================================================
 
-Prepare for Test
+Prepare for Tests
 ----------------------------------------------------------------------
 
-Set the Snowflake connection info in ``parameters.json`` and place it in $HOME:
-<pre><code>
-{
-  "testconnection": {
-    "SNOWFLAKE_TEST_USER":      "&lt;your_user&gt;",
-    "SNOWFLAKE_TEST_PASSWORD":  "&lt;your_password&gt;",
-    "SNOWFLAKE_TEST_ACCOUNT":   "&lt;your_account&gt;",
-    "SNOWFLAKE_TEST_WAREHOUSE": "&lt;your_warehouse&gt;",
-    "SNOWFLAKE_TEST_DATABASE":  "&lt;your_database&gt;",
-    "SNOWFLAKE_TEST_SCHEMA":    "&lt;your_schema&gt;",
-    "SNOWFLAKE_TEST_ROLE":      "&lt;your_role&gt;"
-  }
-}
-</code></pre>
+Specify env variables:
```

Comment:
I noticed, that this file is indeed used in some scripts. Are you sure, that we want to get rid of it in this readme?

Reply:
parameters.json is only used in CI tests. It's not necessary to have this file created for local tests.

Reply:
You can use this file if environment variables are not set. So better to keep it.

Reply:
Parameters.json are only used on CI. For running tests locally they are not used. @sfc-gh-igarish do you think we should keep it in readme?